2-mandatory/9-cheap-diner.js

```diff
@@ -19,7 +19,23 @@ chosenMeal(emptyArray)
 Should give the answer "Nothing :("
 **/
 
-function chooseMeal(mealArray) {}
+function chooseMeal(mealArray) {
+  let lowestPriceMeal = mealArray
+  .map((x) => x.price)
+    .sort((a, b) => a - b);
```

Comment:
Great that you are able to write something with this complexity 👍 

Well done!

There might be an easier way that allows you to sort by field without having to map, then you can retain the meal name in the sorted array.